app/routes/announcements/components/AnnouncementItem.js

```diff
@@ -84,19 +84,22 @@ const AnnouncementItem = ({
         </Flex>
       </Flex>
       {!announcement.sent &&
-        actionGrant.includes('send') && (
+        actionGrant.includes('send') &&
+        actionGrant.includes('delete') && (
           <Flex className={styles.wrapperSendButton}>
             <Button
               className={styles.sendButton}
               onClick={() => deleteAnnouncement(announcement.id)}
             >
-              Slett
+              {' '}
+              Slett{' '}
```

Comment:
Are these spaces necessary? I think they're kinda fine to use in the middle of a sentence where setting a margin just to imitate a space would be akward, like 
```
Han er{' '}
{longFunctionDetermeningHowOldTheUserIs()}
{' '}år
```
but they aren't a good way to imitate margin or padding.